README.md

```diff
@@ -1,15 +1,13 @@
-smoothscroll.js
-=================
+# smoothscroll.js
 
 Polyfill for
-* window.scroll({left, top, behavior});
-* window.scrollTo({left, top, behavior});
-* window.scrollBy({left, top, behavior});
+* window.scroll({ left, top, behavior });
+* window.scrollTo({ left, top, behavior });
+* window.scrollBy({ left, top, behavior });
 
-* element.scrollIntoView(ScrollOptions); (incomplete)
+* element.scrollIntoView(scrollOptions); (incomplete)
```

Comment:
This was intentionally capitalized to reference the `ScrollOptions` object from the spec.